src/player.cpp

```diff
@@ -2097,11 +2097,32 @@ Item* Player::getCorpse(Creature* lastHitCreature, Creature* mostDamageCreature)
 {
 	Item* corpse = Creature::getCorpse(lastHitCreature, mostDamageCreature);
 	if (corpse && corpse->getContainer()) {
+		std::map<std::string, uint16_t> names;
```

Comment:
You don't seem to use the count, so this could be just a `set` (or even better, an `unordered_set`)

Reply:
Your suggestion seems very good, but I don't know how to easily increase the counter.
for example:
```c++
names[killer->getName()]++;
```
^ this is impossible with a `set` or `unordered_set`

and do `names.find(name).second++` It doesn't seem to work, and if it works, I don't know if it will cost more than using the map

Reply:
But you don't use the count for anything, you don't have to keep track of it.